### PR TITLE
No git clean

### PR DESCRIPTION
**Pull request by miigotu**

### CODE REVIEW

**Comment by fernandog** [comment]
@miigotu  why not just do 'git clean -f' instead of -df

Not matter the name of folder tracked files wont be removed

```
-d
Remove untracked directories in addition to untracked files
```

**Comment by miigotu** [comment]
because it is not just folders that are being removed and shouldnt be. There are 0 reasons to do a git clean.

**Comment by fernandog** [comment]
Also backups are not being removed as 'backup' folder is in gitignore and user can change as its hardcoded

**Comment by fernandog** [comment]
If there are files being removed we must add to .gitignore

**Comment by miigotu** [comment]
Backups are being removed. It has been reported tons of times. So ok the backup folder is ignored, what about the zip/rar inside?

There is NO REASON we need to git clean. Just dont git clean and it solves it all at once.

**Comment by miigotu** [comment]
What if a user put a folder containing their video library in the SR dir with some randon name? They would lose all of their videos.

File: gui/slick/views/config_general.mako [hunk 1]
```diff
@@ -729,7 +729,7 @@
                             </label>
                         </div>
 
-                        <div class="field-pair">
+                        <div class="field-pair" hidden>
```
Comment (MGaetan89) [inline gui/slick/views/config_general.mako]: "hidden" is outside of "class" ;)

**Comment by fernandog** [comment]
I understand that. But as explain above: removing the -d it wont remove folders. Just fyi.
The backup feature was added a long time ago and we never had issues but its ok. You dont want git clean. I get that.

**Pull request merged**